### ISSUE TITLE
[TW-1895] Cannot create task with explicit description 'start ....'

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1908]
_Matt Chun-Lum on 2017-02-16T03:14:03Z says:_ 

 `task version`
`2.5.1`

The following works:
`task add 'start something'`

The following does not:
`task add description:'start something'`

Removing the 'start' fixes the issue
`task add description:'something'`

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2017-02-16T03:14:03Z
Modified: 2017-10-12T03:07:01Z
```

**Comment by taskwarrior** [comment]
_Eric Hymowitz on 2017-02-16T17:51:12Z says:_ 

 There have been similar bugs when people try to use what TW views as a "reserved word".  Probably related.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2017-10-12T03:07:01Z says:_ 

 Eric is right.  Good catch.

**Comment by alexwms** [comment]
I am experiencing a similar issue. Executing 
`task rc:~/.taskrc rc.json.array=TRUE rc.verbose=new-uuid rc.confirmation=no rc.dependency.confirmation=no ad72e1e8-b814-4305-941f-43d6759266b1 modify description:"start writing notes" due:"20190729T000000Z" entry:"20190727T162131Z" focus:"0" modified:"20190727T162131Z" status:"pending" tags:"research" urgency:"9.90866"`
 results in
`Configuration override rc.json.array:TRUE `
`Configuration override rc.verbose:new-uuid`
`Configuration override rc.confirmation:no`
`Configuration override rc.dependency.confirmation:no`
`The expression could not be evaluated.`

(The command above modifies the existing task with UUID ad72e1e8-b814-4305-941f-43d6759266b1. This command is used by the function task_update method in the taskw package).

However, running `task rc:~/.taskrc rc.json.array=TRUE rc.verbose=new-uuid rc.confirmation=no rc.dependency.confirmation=no ad72e1e8-b814-4305-941f-43d6759266b1 modify description:"writing notes" due:"20190729T000000Z" entry:"20190727T162131Z" focus:"0" modified:"20190727T162131Z" status:"pending" tags:"research" urgency:"9.90866"`
does not produce any errors.

It seems like this would be a good thing to get to the bottom of since it seems quite common to start the description of a task with the word start. If there is a priority ranking for which bugs will be worked on first, I would like to vote for this one.

**Comment by tbabej** [comment]
Reproduced on 2.5.2, 2.6.0. @pbeckingham This is potentially the only one of the keyword-based issues that is still reproducible, but the error changed from:

> The expression could not be evaluated.
(on 2.5.1)

to:
> The value is not an expression.
(on 2.5.2, 2.6.0)

**Comment by tbabej** [comment]
This would be fixed by #1965.
